gsitemap.php

```diff
@@ -279,7 +279,23 @@ public function _addLinkToSitemap(&$link_sitemap, $new_link, $lang, &$index, &$i
                 if ($this->cron) {
                     Tools::redirectAdmin($this->context->link->getBaseLink().'modules/gsitemap/gsitemap-cron.php?continue=1&token='.Tools::substr(Tools::encrypt('gsitemap/cron'), 0, 10).'&type='.$new_link['type'].'&lang='.$lang.'&index='.$index.'&id='.(int)$id_obj.'&id_shop='.$this->context->shop->id);
                 } else {
-                    Tools::redirectAdmin($this->context->link->getBaseLink().'index.php?tab=AdminModules&configure=gsitemap&token='.Tools::getAdminTokenLite('AdminModules').'&tab_module='.$this->tab.'&module_name=gsitemap&continue=1&type='.$new_link['type'].'&lang='.$lang.'&index='.$index.'&id='.(int)$id_obj.'&id_shop='.$this->context->shop->id);
+					Tools::redirectAdmin(
```

Comment:
Watch out, you used Tabs instead of Spaces :)

Reply:
Yes, sorry for thank. I saw it. I will fix it soon :-)

Reply:
Fixed